lib/tools/mock-module.ts

```diff
@@ -40,11 +49,11 @@ export function mockModule<TModule extends AnyNgModule>(mod: TModule, setup: Tes
 
   const modClass = mod as Type<any>;
 
-  const {imports, declarations, exports, entryComponents, providers, schemas} = getNgModuleAnnotations(modClass);
+  const {imports, declarations, exports: exp, entryComponents, providers, schemas} = getNgModuleAnnotations(modClass);
```

Comment:
Is this rename intentional? Feel free to ignore this if this was meant to be renamed.

Reply:
Nice catch.. Fixed.